### DISCUSSION TITLE
HTML compatible stream using ffmpeg

### DISCUSSION THREAD

**Discussion by baudev** [gtxaspec/wz_mini_hacks#400] [Development]
I want to display the video stream on an HTML webpage accessible outside the local network (with a low latency).

However, I don't want to use HLS as writing continuously segments to the SD card would damage it. I've already destroyed several cards like this.  
Also I would like the stream to be generated only by the camera. I don't want to use an outside RTMP server for example.

Maybe with `ffmpeg` that would be possible?

Thanks for your suggestions

**Comment by Bob-AOL** [comment]
OBS?  https://obsproject.com/
